front/components/welcome/WelcomeThird.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-content>
+    <welcome-desktop
+      class="hidden-mobile"
+      :params="params"
+      :hide-next-btn="true"
```

Comment:
still same hide nxt btn

Reply:
I missed the other ones for some reason. Now it should be pushed.